### ISSUE TITLE
SNTP check may no longer serve it's original purpose?

### ISSUE THREAD

**Issue by mdavids** [mlichvar/ntptraf#1]
I was wondering if the [SNTP-check](https://github.com/mlichvar/ntptraf/blob/a1e1af3bd2992db9d858aa2c7f27abd8c2e0e2bb/ntptraf.c#L173) still makes sense in the light of the draft mentioned below?

https://datatracker.ietf.org/doc/html/draft-ietf-ntp-data-minimization

https://github.com/mlichvar/ntptraf/blob/a1e1af3bd2992db9d858aa2c7f27abd8c2e0e2bb/ntptraf.c#L173

**Comment by mlichvar** [comment]
No, it does not make much sense, even before that draft it wasn't very reliable. It just checks for a non-zero origin timestamp. There might be a better description for this statistics.
